timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md

```diff
@@ -274,18 +332,24 @@ When you have created the certificates and keys, and moved all the files into th
 <procedure>
 
 ### Configuring data nodes to use SSL authentication
+
 1.  On each data node, open the `postgresql.conf` configuration file and add or
     edit the SSL settings to enable certificate authentication:
+
     ```txt
     ssl = on
     ssl_ca_file = 'root.crt'
     ssl_cert_file = 'server.crt'
     ssl_key_file = 'server.key'
     ```
-    **OPTIONAL:** If you want the access node to use certificate authentication for login, make these changes on the access node as well.
+
+1. <optional />If you want the access node to use certificate authentication
```

Comment:
This line looks like it needs another whitespace char.

Reply:
done